lib/animas/animasSimulator.js

```diff
@@ -261,14 +265,23 @@ exports.make = function(config){
               }
             }
           }
+
+          // Filter out dates 2008 and earlier, as this is year 0 for Vibe.
+          // We are including 2008 as it's possible that the pump ran for a
```

Comment:
I meant that we're including 2008 in the filter - I understand how that can be misunderstood, will change. And yes, the rest of your explanation is correct, so I'll include it as part of the comments for future reference (parts of it verbatim if you don't mind ;)